### PR TITLE
upgrade clusterlifecycle-state-metrics versions

### PR DESCRIPTION
**Pull request by haoqing0110**
Ref: https://github.com/open-cluster-management/backlog/issues/17931
https://github.com/open-cluster-management/backlog/issues/15570
Signed-off-by: haoqing0110 <qhao@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-main-postsubmits.yaml [hunk 1]
```diff
@@ -14,9 +14,9 @@ postsubmits:
       containers:
       - args:
         - --config-dir=../../openshift/release/ci-operator/config/open-cluster-management/clusterlifecycle-state-metrics/
-        - --current-release=2.4
```
Comment (zhiweiyin318) [inline ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-main-postsubmits.yaml:17]: suggest use ocm-ci-fastforward https://github.com/openshift/release/blob/19718ba06831c899f80e9a563403ff5cfa72f0ca/ci-operator/config/open-cluster-management/helloprow-go/open-cluster-management-helloprow-go-main.yaml#L104 to do ff

Comment (haoqing0110) [reply ci-operator/jobs/open-cluster-management/clusterlifecycle-state-metrics/open-cluster-management-clusterlifecycle-state-metrics-main-postsubmits.yaml:17]: @zhiweiyin318 thx, fixed.

**Pull request merged**